.ci_scripts/run_cmake.sh

```diff
@@ -10,6 +10,8 @@ case "$BUILD_TARGET" in
 "mac")
 	mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release -DSYSTEM_LIBS=OFF -DCMAKE_OSX_ARCHITECTURES="x86_64;arm64" ..
 	;;
+"flatpak")
```

Comment:
How is cmake invoked for Flatpak?

Reply:
`flatpak-builder repo com.github.bvschaik.julius.json` in `run_build.sh` will call cmake (with ninja instead of make) as it is defined as `"buildsystem" : "cmake-ninja"`